src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { FullPost } from '../../types/FullPost';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type ListOfpostProps = {
```

Comment:
Consider renaming 'ListOfpostProps' to 'ListOfPostProps' for better readability and consistency in naming conventions. 🧐